cloudify_docker_image/README.md

```diff
@@ -28,7 +17,7 @@ network adapter's mtu size and edit the docker service file with
 ```
 # The path also might be at /usr/lib/systemd/system/docker.service
 sudo vi /etc/systemd/system/docker.service
-# At 'ExecStart=' add '--mtu HOST_MTU_SIZE' flag to the end of the line
+# At 'ExecStart=' add '--mtu <HOST_MTU_SIZE>' flag to the end of the line
```

Comment:
Can you do this with sed?

Reply:
I could but I don't think it's worth the time (gotta get the correct interface mtu and then replace it in the file)